src/components/Filters/Filters.tsx

```diff
@@ -535,4 +543,9 @@ function getShortcutFilters(filters: FilterInterface[]) {
   return filters.filter((filter) => filter.shortcut === true);
 }
 
-export const Filters = withAppProvider<FiltersProps>()(FiltersInner);
+function FiltersInnerWrapper(props: ComposedProps) {
```

Comment:
Had to add this since `contextType` is already being used and we removed the `useContext` HoC